src/components/modal/modal.module.css

```diff
@@ -16,13 +16,13 @@
   left: 50%;
   border-radius: var(--border-radius-2);
   box-shadow: var(--box-shadow-modal);
-  opacity: 0;
+  visibility: hidden;
   pointer-events: none;
   transform: translate(-50%, -50%);
   transition-property: opacity;
 }
 .open.center {
-  opacity: 1;
+  visibility: visible;
```

Comment:
should not it display none instead ?

Reply:
Hm good question, dunno, will try

Reply:
"Should" is strong word, dunno if it's really better as CSS transitions don't apply with display: none switching
But I should have kept opacity for the same reason, thanks for pointing it out

Reply:
I don't know tbh :) but display have different behaviour vs vibility

Reply:
ok, kept visibility for now for the aforementioned transition to work